vutil/vutil.h

```diff
@@ -151,33 +157,33 @@ static const char * Perl_prescan_version2(pTHX_ const char *s, bool strict, cons
 
 #else
 
-const char * Perl_scan_version(pTHX_ const char *s, SV *rv, bool qv);
```

Comment:
This is plain wrong.

It would be silly for ordinary dists, but it's exactly right for core, and given that this is shared with core it should remain like it is.